circuits/environment/src/lib.rs

```diff
@@ -15,6 +15,7 @@
 // along with the snarkVM library. If not, see <https://www.gnu.org/licenses/>.
 
 #![forbid(unsafe_code)]
+#![allow(clippy::too_many_arguments)]
```

Comment:
Was there a specific case that required this clippy lint?

Reply:
Used for `test_utilities`. No longer needed.